src/main/java/com/synopsys/integration/alert/channel/jira/common/JiraTestIssueRequestCreator.java

```diff
@@ -77,14 +77,12 @@ public IssueTrackerRequest createRequest(IssueOperation operation, String messag
 
             switch (operation) {
                 case RESOLVE: {
-                    IssueContentModel contentModel = jiraMessageParser.createIssueContentModel(providerName, IssueResolutionRequest.OPERATION, topicItem, subTopicItem, componentItems, arbitraryItem);
-                    return IssueResolutionRequest.of(issueSearchProperties, contentModel);
+                    return createResolveIssueRequest(providerName, topicItem, subTopicItem, componentItems, arbitraryItem, issueSearchProperties);
```

Comment:
SonarCloud does not like code in the switch cases, to make the cases easier to understand moved the code to a method